package.json

```diff
@@ -31,5 +31,6 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
-    }
+    },
```

Comment:
I don't know what this does to the larger project but I would revert this just to be safe.

Reply:
sorry about that. npm automatically added it when I ran `npm install`